packages/elements-core/src/__fixtures__/articles/kitchen-sink.md

```diff
@@ -211,6 +211,14 @@ be the json schema object to be rendered.
               "type": "number",
               "minimum": 0,
               "maximum": 150
+            },
+            "type": {
+              "type": "string",
+              "enum": ["STANDARD", "ADMIN"],
+              "x-enum-descriptions": {
```

Comment:
Updated a bunch of fixtures, like this, one to accommodate for testing vendor extensions functionality both in Storybook and unit tests